etc/migration/disable_users_from_tenancy/lib/disable-users-by-tenancy.js

```diff
@@ -29,6 +37,9 @@ var PrincipalsAPI = require('oae-principals');
  * @param  {Function}   callback        Standard callback function
  */
 var doMigration = function (ctx, tenantAlias, disabled, callback) {
+
+    ctx = ctx || _createNewContext();
```

Comment:
Well, `_createNewContext` generates a new global admin context which would be not-good anywhere except in a migration script, OK here though.